checks/evaluation/maintained.go

```diff
@@ -30,69 +33,72 @@ const (
 )
 
 // Maintained applies the score policy for the Maintained check.
-func Maintained(name string, dl checker.DetailLogger, r *checker.MaintainedData) checker.CheckResult {
-	if r == nil {
-		e := sce.WithMessage(sce.ErrScorecardInternal, "empty raw data")
-		return checker.CreateRuntimeErrorResult(name, e)
+func Maintained(name string,
+	findings []finding.Finding, dl checker.DetailLogger,
+) checker.CheckResult {
+	// We have 4 unique probes, each should have a finding.
+	expectedProbes := []string{
+		notArchived.Probe,
+		issueActivityByProjectMember.Probe,
+		commitsInLast90Days.Probe,
```

Comment:
I thought the `90` days would be part of the finding's `Value` and not the name of the probe?

Reply:
The downside to that is that if we generalize the name, we might have issues when we want to change the duration-parameter (the 90 days) later for other probes. For example, if we call the probe `numberOfRecentCommits` which is tied to 90 days, how do we create probes later with 30 days or 6 months?

@spencerschrock What is your opinion?

Reply:
I believe we said this was similar to `--commit-depth`, where the user customizes the behavior. I was also in favor of dropping 90 from the name but wasn't going to block on it. I think we said we were going to add the lookback period as a value though.

Something like `"lookback": 90`

I assume we'll be doing a final review on names/descriptions/etc before release.

Reply:
if it's easy to fix in this PR, I vote for doing it. Or let's create a tracking issue pointing to this comment

Reply:
I have created https://github.com/ossf/scorecard/issues/3665 to track that.